site/content/en/guides/consumer/function/export/jenkins/_index.md

```diff
@@ -54,13 +63,13 @@ curl -fsSL https://get.docker.com -o get-docker.sh
 sudo sh get-docker.sh
 ```
 
-1.  Add the `jenkins` user to the `docker` group so that docker commands can be run in Jenkins pipelines.
+2.  Add the `jenkins` user to the `docker` group so that docker commands can be run in Jenkins pipelines.
```

Comment:
Undo these changes, it's best practice to keep "1." for all steps.

Reply:
Here is the only case that they are not rendered correctly.

Reply:
It is because the code blocks. I will fix them.